### ISSUE TITLE
デスクトップの番号をタスクトレイに表示するの時の常駐アイコンがライトテーマだと見づらい

### ISSUE THREAD

**Issue by juner** [Grabacr07/SylphyHorn#50]
`デスクトップの番号をタスクトレイに表示する` を ON にした際、
ライトテーマだと
![image](https://user-images.githubusercontent.com/753914/87102836-31827800-c28e-11ea-8a26-488204e22c87.png)
の様に表示されて可読性がゼロになります。

ちなみにダークテーマのときは次の様な感じです。
![image](https://user-images.githubusercontent.com/753914/87102914-6db5d880-c28e-11ea-92a2-7a2b370e4c15.png)

**Comment by EtiamNullam** [comment]
I agree, its hard to read on light background. I always use black taskbar so it never was an issue for me.

I think we should decide and do one of these:
- Draw black text when taskbar color is too bright (what to do if its transparent?)
- Draw black circle or rounded square before drawing text
- Draw thick outline around text, just like video players do it for subtitles
